### PR TITLE
Add Yahoo Japan case study

### LINKED ISSUE #5807: [Yahoo Case Study - English] - Build takeover and engage page
## Requirements

Please build an engage page based on this [brief](https://docs.google.com/spreadsheets/d/1txCEDcgszYy43FEnxEc2lV3EG6vcYpJSnO878T-s5VU/edit#gid=1595677042) and [design](1693):
 

- [x] build engage page
- [x] add engage page to `/engage/index.html`
- [ ] test the engage page on [https://socialdebug.com/](https://socialdebug.com/) or [http://debug.iframely.com/](http://debug.iframely.com/) and [https://cards-dev.twitter.com/validator](https://cards-dev.twitter.com/validator)

_Note; as this is a case study there is no takeover._

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- New Yahoo! Japan case study engage page.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/yahoo-japan-case-study and http://0.0.0.0:8001/engage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare it to the [brief](https://docs.google.com/spreadsheets/d/1txCEDcgszYy43FEnxEc2lV3EG6vcYpJSnO878T-s5VU/edit#gid=1595677042) and the [copy doc](https://docs.google.com/document/d/1WHnbQO6m6B4puJr8Vc3WxPC9BBJ0E7pvkTXSJUKvCM0/edit)
- Test the form

## Issue / Card

Fixes #5807

### CODE REVIEW

File: templates/engage/yahoo-japan-case-study.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<blockquote class="p-pull-quote">
+  <p class="p-pull-quote__quote">During unexpected incidents, the cause needs to be tracked down to the OS layer and being able to ask a professional is a big advantage. I think Canonical is the only company capable of offering this robust support,</p>
+  <cite class="p-pull-quote__citation">Mr. Sakata, Cloud Development Leader, Yahoo! Japan</cite>
+</blockquote>
+
+Using Ubuntu and a wide range of Canonical’s management tools, Yahoo! Japan has been able to reduce both CAPEX and OPEX costs while successfully building and operating a large scale IaaS environment.
+
+In this case study, learn how Yahoo! Japan:
+
+- Deployed an [OpenStack](/openstack) private cloud with 1PB of Ceph storage across three regions using <a class="p-link--external" href="https://maas.io">MaaS</a> and <a class="p-link--external" href="https://jaas.ai">Juju</a>.
```
Comment (anthonydillon) [inline templates/engage/yahoo-japan-case-study.md]: MAAS is capitalised.

File: templates/engage/yahoo-japan-case-study.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- Benefited from the reassurance and expertise of Canonical engineers through <a class="p-link--external" href="https://buy.ubuntu.com/">Ubuntu Advantage</a> to ensure unexpected incidents are kept to a minimum.
+- Have future-proofed themselves by taking advantage of ESM (Extended Security Maintenance) for security patches until April 2028.
```
Comment (anthonydillon) [inline templates/engage/yahoo-japan-case-study.md]: Should we link to the esm page?

File: templates/engage/yahoo-japan-case-study.md (hunk 1 above)
Comment (pmahnke) [reply templates/engage/yahoo-japan-case-study.md]: good catch

File: templates/engage/yahoo-japan-case-study.md (hunk 2 above)
Comment (pmahnke) [reply templates/engage/yahoo-japan-case-study.md]: sure, also found a bad favicon!

**Comment by anasereijo** [comment]
@pmahnke looks good

**Pull request merged**